### ISSUE TITLE
Exception if Parameters values in posted forms exceed 1024bytes 

### ISSUE THREAD

**Issue by marcban** [eclipse-vertx/vert.x#3834]
### Questions
I get an IOException when handling long attributes values in POSTed forms, which I had not on 3.9

### Version
Vertx core/web 4.0.2

### Context

When using BodyHandler I get an IOException on long parameters values
"IOException Size exceed allowed maximum capacity"
This seems to be a regression, I had not the problem on 3.9.X

### Do you have a reproducer?

No but offending line seems to be on file NettyFileUploadDataFactory, line 82 : 
```
 private static class VertxAttribute extends MemoryAttribute {
    public VertxAttribute(String name, long definedSize) {
      super(name, definedSize);
      setMaxSize(1024); //// here the netty attribute is limited in size, but not the definedSize
    }
```

**Comment by marcban** [comment]
@vietj I see this is already corrected in #3826

Please make a release soon :-) !

**Comment by marcban** [comment]
Duplicate of #3794
